num2words/lang_ES_MX.py

```diff
@@ -0,0 +1,35 @@
+# encoding: UTF-8
+
+# Copyright (c) 2003, Taro Ogawa.  All Rights Reserved.
+# Copyright (c) 2013, Savoir-faire Linux inc.  All Rights Reserved.
+
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301 USA
+
+from __future__ import print_function, unicode_literals
+
+from .currency import parse_currency_parts
```

Comment:
Buen dia Salvador, estoy tratando de probar este nuevo cambio pero al importar la clase me marca el siguiente error
`File "/opt/pyerp_master/eggs/num2words-0.5.3-py2.7.egg/num2words/lang_ES_MX.py", line 21, in <module>
   from .currency import parse_currency_parts
ImportError: No module named currency`
Ya instale un modulo que se llama currency `bin/pip install currency` pero aun asi me sigue saliendo el mismo error, como se llama la libreria que usas tu para poder importar la funcion `parse_currency_parts`
Saludo

Reply:
Saludos Alfonso, ese currency se refiere al archivo currency.py que esta en la raiz del mismo directorio:

https://github.com/savoirfairelinux/num2words/blob/master/num2words/currency.py

No es un módulo o librería externa.

Saludos!

Reply:
Saludos Salvador, pense que era una libreria externa ya que en la version que tengo en mi instancia de la libreria num2words no viene la clase currency.py, muchas gracias por la informacion y apoyo
Saludos!!